bloom/commands/release.py

```diff
@@ -680,20 +680,28 @@ def get_gl_info(url):
 _gl = None
 
 
-def get_github_interface(quiet=False):
-    global _gh
-    if _gh is not None:
-        return _gh
-    # First check to see if the oauth token is stored
+def get_bloom_config_and_path():
     oauth_config_path = os.path.join(os.path.expanduser('~'), '.config', 'bloom')
     config = {}
     if os.path.exists(oauth_config_path):
         with open(oauth_config_path, 'r') as f:
             config = json.loads(f.read())
-            token = config.get('oauth_token', None)
-            username = config.get('github_user', None)
-            if token and username:
-                return Github(username, auth=auth_header_from_oauth_token(token), token=token)
+    return config, oauth_config_path
+
+
+
```

Comment:
Is this one too many line breaks?

Reply:
ERROR: Too many linting errors for me to notice this one.

Thanks.